README.md

```diff
@@ -71,6 +71,7 @@ Once this is done, you are ready to configure any of the available strategy in t
 
 ### Oauth2-client-password
 
+In order to use it, run `npm install passport-oauth2-client-password`.
```

Comment:
Also mention about the specific provider binding required

Reply:
resolved